### PR TITLE
[readme-api][s]: added API section in README.md - refs #3

### PR DESCRIPTION
**Pull request by Mikanebu**
* added API section in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,3 +15,33 @@ Resolver is a DataHub microservice for resolving datapackage URLs into more huma
...
+`/resolver/resolve`
+
+**Method:** `GET`
+
+**Query Parameters:**
+
+ - `path` - takes {username/pkgname}
+
+**Returns:**
+If exists username and pkgname:
```
Comment (zelima) [inline README.md]: if user found in the system:

File: README.md [hunk 2]
```diff
@@ -15,3 +15,33 @@ Resolver is a DataHub microservice for resolving datapackage URLs into more huma
...
+```json
+{
+    "packageid": "<packageid>",
+    "userid": "<userid>"
+}
+```
+if there is no user, it return null:
```
Comment (zelima) [inline README.md]: If user not found

File: README.md [hunk 3]
```diff
@@ -15,3 +15,33 @@ Resolver is a DataHub microservice for resolving datapackage URLs into more huma
 # Run server
 
 `python server.py`
+
+
+## API
+
+### Get information about userid and packageid
```
Comment (zelima) [inline README.md]: I don't think this is what it does. 
something like "resolves username to userid" would be better. Or maybe you can think something better

**Review by zelima — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (Mikanebu) [reply README.md]: I like your suggestion, it is short and clear

File: README.md [hunk 4]
```diff
@@ -15,3 +15,33 @@ Resolver is a DataHub microservice for resolving datapackage URLs into more huma
...
+`/resolver/resolve`
+
+**Method:** `GET`
+
+**Query Parameters:**
+
+ - `path` - takes {username/pkgname}
+
+**Returns:**
+if user found in the system::
```
Comment (zelima) [inline README.md]: * Capital I
* extra colon

Comment (Mikanebu) [reply README.md]: done

**Pull request merged**